lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -125,10 +129,15 @@ private static IConfiguration GetRuntimeConfiguration(string name)
                         .Set(AzureBatchRuntimeClientConfiguration.AzureStorageAccountName, @"############")
                         .Set(AzureBatchRuntimeClientConfiguration.AzureStorageContainerName, @"###########")
                         //// Extend default retry interval in Azure Batch
-                        .Set(AzureBatchRuntimeClientConfiguration.DriverHTTPConnectionRetryInterval, "20000")
+                        .Set(AzureBatchRuntimeClientConfiguration.DriverHTTPConnectionRetryInterval, "2000")
```

Comment:
will 2 second be longer enough? In my experience, it may timeout and throw exception with 2 second retry. I think you might stayed too long in break point so you didn't hit this error.

Reply:
Sorry, that was in error. Removed.

---
In reply to: [200411933](https://github.com/sharathmalladi/reef/pull/6#discussion_r200411933) [](ancestors = 200411933)